makefile

```diff
@@ -152,14 +152,29 @@ build-mpi: $(MPI_TARGETS)
 	@echo '--- boost mpi bindings built ---'
 
 .PHONY: build
-build: bin/stanc$(EXE) bin/stansummary$(EXE) bin/print$(EXE) bin/diagnose$(EXE) $(LIBSUNDIALS) $(MPI_TARGETS) $(CMDSTAN_MAIN_O)
-	@echo ''
+build: bin/stanc$(EXE) bin/stansummary$(EXE) bin/print$(EXE) bin/diagnose$(EXE) $(LIBSUNDIALS) $(MPI_TARGETS) $(TBB_TARGETS) $(CMDSTAN_MAIN_O)
+	@echo ''
+	@if [ "$(OS)" == "Windows_NT" ]; then \
```

Comment:
Argh, I found an ugly warning that doesnt really prevent anything, the thing still builds. Its just ugly.
On ubuntu I get

`/bin/sh: 1: [: Linux: unexpected operator`

Mind changing this to 

```
.PHONY: build
build: bin/stanc$(EXE) bin/stansummary$(EXE) bin/print$(EXE) bin/diagnose$(EXE) $(LIBSUNDIALS) $(MPI_TARGETS) $(TBB_TARGETS) $(CMDSTAN_MAIN_O)
	@echo ''
ifeq ($(OS),Windows_NT)
		echo 'NOTE: Please add $(TBB_BIN_ABSOLUTE_PATH) to your PATH variable.';
		echo 'You may call';
		echo '';
		echo 'mingw32-make install-tbb';
		echo '';
		echo 'to automatically update your user configuration.';
endif
	@echo '--- CmdStan v$(CMDSTAN_VERSION) built ---'
```

Reply:
sure.